switchmap/poller/async_poll.py

```
@@ -0,0 +1,299 @@
+                log_message = (
+                    f"Device {hostname} returns no data. Check "
+                    f"connectivity/SNMP configuration"
+                )
+                log.log2debug(1408, log_message)
+                return False
+
+        except (asyncio.TimeoutError, KeyError, ValueError) as e:
+            log_message = f"Recoverable error polling device {hostname}: {e}"
+            log.log2warning(1409, log_message)
+            return False
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)